go.mod

```
@@ -1,6 +1,6 @@
 module sigs.k8s.io/controller-tools
 
-go 1.24.0
+go 1.24.2
```

```suggestion
go 1.24.0
```
Let's keep this version in go.mod at .0 please. I do not want to force folks to use a specific patch version when they import controller-tools

---

Thanks @sbueringer 
Revert it done.